proposals/stage-1/SUPPORTER_PROGRAM/SUPPORTER_PROGRAM.md

```diff
@@ -0,0 +1,51 @@
+## The People's Community of JavaScriptlandia 
+
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+It's an Individual Membership Program whose goals are to help bring awareness about OpenJS projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community with the other members. 
+
+While one goal of this group is introducing members to projects and helping the projects find the support they need; the membership program will also be an opportunity to welcome an inclusive group of coders from around the world that love the projects that are part of the OpenJS foundation. This also brings in new perspectives and feedback from the vast community of JavaScript developers.
+
+We also want to grow awareness of OpenJS, our projects, and how to get involved. 
+
+Additionally, Individual membership should help people understand that their projects too can get support from the foundation, and the OpenJS foundation is their foundation. 
+
+A tertiary goal is to establish a pool from which an ambassador program can be started. Recognizing people that evangelize and recruit for OpenJS projects. 
+
+## Membership Benefits
+
+Citizens of JavaScriptlandia will receive a members weekly newsletter keeping them up to date on the last from OpenJS projects, the CPC, and the board. They will be invited to participate in discussions about governance and new initiatives. 
+
+They will be featured on our global membership page, and will be given a digital passport that they can put on an avatar, their blog, and/or personal website. Their passport will feature the libaries that they use the most, and the ones that are the biggest fans of. 
+
+There will be a repository for members, and they will be able to display their badge proudly on their Github profile. Members can incubate projects in that repository (this should have discussion).
+
+Additionally, they will receive discounts for training, certification, conferences, and potentially exclusive offers from member organizations. 
+
+Lastly, members will have access to the member store where they can purchase OpenJS member swag to show their support for our projects. Revenue from swag will go towards furthering the reach of the foundation. 
+
+
+## Membership Cost
+
+Membership is charged at $25/year. Membership is renewed 1 year from the date of your membership. Members can cancel at any time and not be charged for the next year. 
```

Comment:
There should be discussion about the automatic renewal with the staff that will be maintaining memberships and payments. In the past when running the individual membership program for the jQuery Foundation, there was a lot of time spent dealing with refund requests when folks forget to cancel, expired/invalid cards that can't renew, etc. I think they would want to weigh auto-renewal against perhaps automated reminders to manually renew and how each will impact staff toil as well as member retention.

I'm not saying auto-renewal is the wrong way to go, I just want to make sure those that will be handling the process are looped in so we make a good decision.

Reply:
This is a good note, thank you. There are pros and cons to each. I'll add this to the list of things to discuss.